interfaces/cython/cantera/kinetics.pyx

```diff
@@ -63,6 +63,9 @@ cdef class Kinetics(_SolutionBase):
     of progress, species production rates, and other quantities pertaining to
     a reaction mechanism.
     """
+    #: List holding references to CustomRate objects (prevents garbage collection)
+    _custom_rates = []
```

Comment:
This makes sense, does Cython have a destructor method for classes? It might make sense to clear this list when the Kinetics instance is GCed if you can do it in the destructor

Reply:
I believe this happens automatically.